style-guide.rst

```diff
@@ -520,32 +543,72 @@ A big advantage of reST in comparison to plain Markdown is that it allows to reu
 Substitution
 ~~~~~~~~~~~~
 
-To reuse sentences or paragraphs without too much markup and special formatting, use substitutions.
+To reuse sentences and entire paragraphs
+that have little markup or special formatting,
+define their `substitutions`_ in two possible ways.
+
+**Globally**, in a file named :file:`reuse/substitutions.txt`
+that is included in the `rst_epilog`_ directive:
```

Comment:
How about:

```suggestion
that is included in the ``rst_epilog`` directive (see `Sphinx documentation <rst_epilog_>`_:
```

This would avoid the need for `rst` in the word list.